cmd/root.go

```diff
@@ -709,16 +440,18 @@ var jCommentIDRegex = regexp.MustCompile("^Comment \\(ID (\\d+)\\)")
 // createCommments takes a GitHub issue, and retrieves all of its comments. It then
 // matches each one to a comment in `existing`. If it finds a match, it calls
 // updateComment; if it doesn't, it calls createComment.
-func createComments(ghIssue github.Issue, jIssue jira.Issue, existing []jira.Comment, ghClient github.Client, jClient jira.Client) error {
+func createComments(config lib.Config, ghIssue github.Issue, jIssue jira.Issue, existing []jira.Comment, ghClient github.Client, jClient jira.Client) error {
```

Comment:
same: all of these functions that are *not* directly concerned with configuring and launching the command should be outside of this file and package. this function should be in lib.